eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringMongoReader.java

```diff
@@ -83,25 +79,35 @@ private SubmittedVariantEntity getSubmittedVariantEntity(Document notClusteredSu
 
     @Override
     public void open(ExecutionContext executionContext) throws ItemStreamException {
-        MongoDatabase db = mongoClient.getDatabase(database);
-        MongoCollection<Document> collection = db.getCollection(SUBMITTED_VARIANT_ENTITY);
         Bson query = Filters.and(Filters.in(ASSEMBLY_FIELD, assembly),
                 Filters.exists(CLUSTERED_VARIANT_ACCESSION_FIELD, readOnlyClusteredVariants));
         logger.info("Issuing find: {}", query);
-        FindIterable<Document> notClusteredSubmittedVariants = collection.find(query)
-                                                                         .noCursorTimeout(true)
-                                                                         .batchSize(chunkSize);
-        cursor = notClusteredSubmittedVariants.iterator();
+
+        FindIterable<Document> notClusteredSubmittedVariantsDbsnp =
```

Comment:
Just to check my understanding: these are sometimes _clustered_ submitted variants depending on which step we're executing and whether `readOnlyClusteredVariants` is true, is that right?

Reply:
Addressed.